clean_project/search.py

```diff
@@ -0,0 +1,53 @@
+"""
+Search a word in a file and return all the occurrence specified the line numbers
+"""
+import argparse
+import json
+
+BOLD = '\033[1m'
+END = '\033[0m'
+
+
+class Search(object):
+    def __init__(self, json_path):
+        with open(json_path) as json_file:
+            index = json.load(json_file)
+        self.index = index
+
+    def search(self, search_word):
+        """
+        Search if a given word is in bucket of files
+        :param search_word: Word given to search
+        :return: List of files and numbers of file when search_word occurrence
+        """
+        result_list = []
+        if search_word in self.index:
+            result_list = self.index[search_word].items()
+        return result_list
```

Comment:
this function would be simpler by doing:
`return self.index.get(search_word, {}).items()`